{"Level":3,"Created":"2018-08-01 13:27:21.794","Description":"Method:TestConnectCommand","Params":[],"ExceptionMsg":"指定了无效的 IP 地址。   在 System.Net.IPAddress.InternalParse(String ipString, Boolean tryParse)\r\n   在 System.Net.IPAddress.Parse(String ipString)\r\n   在 SAEA.Sockets.Core.BaseClientSocket..ctor(IContext context, String ip, Int32 port, Int32 bufferSize) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Base\\Core\\BaseClientSocket.cs:行号 110\r\n   在 SAEA.RedisSocket.Net.RConnection..ctor(Int32 bufferSize, String ip, Int32 port) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Net\\RConnection.cs:行号 39\r\n   在 SAEA.RedisSocket.Core.RedisConnection..ctor(String ipPort, Boolean debugMode) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Core\\RedisConnection.cs:行号 69\r\n   在 SAEA.RedisSocket.RedisClient..ctor(RedisConfig config, Boolean debugModel) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\RedisClient.cs:行号 61\r\n   在 SAEA.RedisSocket.RedisClient..ctor(String connectStr, Boolean debugModel) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\RedisClient.cs:行号 64\r\n   在 RedisClient.ConnectServerDialogViewModel.<>c__DisplayClass21_0.<ExecuteConnect>b__0() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\ConnectServerDialogViewModel.cs:行号 103\r\n   在 System.Threading.Tasks.Task.InnerInvoke()\r\n   在 System.Threading.Tasks.Task.Execute()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.ConnectServerDialogViewModel.<ExecuteConnect>d__21.MoveNext() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\ConnectServerDialogViewModel.cs:行号 101\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.ConnectServerDialogViewModel.<<get_TestCommand>b__20_0>d.MoveNext() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\ConnectServerDialogViewModel.cs:行号 80","FromIP":"No IP Address"}
{"Level":3,"Created":"2018-08-01 14:10:11.496","Description":"Method:DispatcherUnhandledException","Params":[],"ExceptionMsg":"redis server reply time out!   在 SAEA.RedisSocket.Core.RedisCoder.GetRedisReply(Int32 timeOut) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Core\\RedisCoder.cs:行号 84\r\n   在 SAEA.RedisSocket.Core.RedisCoder.Decoder() 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Core\\RedisCoder.cs:行号 120\r\n   在 SAEA.RedisSocket.Core.RedisOperator.Do(RequestType type, String key) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Core\\RedisOperator.cs:行号 59\r\n   在 SAEA.RedisSocket.RedisClient.Select(Int32 dbIndex) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\RedisClient.cs:行号 158\r\n   在 RedisClient.RedisClientViewModel.<AddDbNodes>b__13_0() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 53\r\n   在 System.Threading.Tasks.Task.InnerInvoke()\r\n   在 System.Threading.Tasks.Task.Execute()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<AddDbNodes>d__13.MoveNext() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 46\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<<get_ConnectCommand>b__30_0>d.MoveNext() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 111\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)\r\n   在 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)\r\n   在 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)","FromIP":"No IP Address"}
{"Level":3,"Created":"2018-08-01 15:59:46.928","Description":"Method:DispatcherUnhandledException","Params":[],"ExceptionMsg":"指定了无效的 IP 地址。   在 System.Net.IPAddress.InternalParse(String ipString, Boolean tryParse)\r\n   在 System.Net.IPAddress.Parse(String ipString)\r\n   在 SAEA.Sockets.Core.BaseClientSocket..ctor(IContext context, String ip, Int32 port, Int32 bufferSize) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Base\\Core\\BaseClientSocket.cs:行号 110\r\n   在 SAEA.RedisSocket.Net.RConnection..ctor(Int32 bufferSize, String ip, Int32 port) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Net\\RConnection.cs:行号 39\r\n   在 SAEA.RedisSocket.Core.RedisConnection..ctor(String ipPort, Boolean debugMode) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Core\\RedisConnection.cs:行号 69\r\n   在 SAEA.RedisSocket.RedisClient..ctor(RedisConfig config, Boolean debugModel) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\RedisClient.cs:行号 61\r\n   在 SAEA.RedisSocket.RedisClient..ctor(String connectStr, Boolean debugModel) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\RedisClient.cs:行号 64\r\n   在 RedisClient.RedisClientViewModel.<>c.<ExecuteConnect>b__31_0() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 139\r\n   在 System.Threading.Tasks.Task`1.InnerInvoke()\r\n   在 System.Threading.Tasks.Task.Execute()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<ExecuteConnect>d__31.MoveNext() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 137\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<<get_ConnectCommand>b__30_0>d.MoveNext() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 126\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)\r\n   在 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)\r\n   在 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)","FromIP":"No IP Address"}
{"Level":3,"Created":"2018-08-01 16:10:20.615","Description":"Method:DispatcherUnhandledException","Params":[],"ExceptionMsg":"redis server reply time out!   在 SAEA.RedisSocket.Core.RedisCoder.GetRedisReply(Int32 timeOut) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Core\\RedisCoder.cs:行号 84\r\n   在 SAEA.RedisSocket.Core.RedisCoder.Decoder() 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Core\\RedisCoder.cs:行号 120\r\n   在 SAEA.RedisSocket.Core.RedisOperator.Do(RequestType type, String key) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\Core\\RedisOperator.cs:行号 59\r\n   在 SAEA.RedisSocket.RedisClient.Select(Int32 dbIndex) 位置 D:\\RedisClientForPc\\RedisClientLibs\\Redis\\RedisClient.cs:行号 158\r\n   在 RedisClient.RedisClientViewModel.<get_ConnectCommand>b__30_0() 位置 D:\\RedisClientForPc\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 133\r\n   在 RedisClient.RelayCommand.Execute(Object parameter) 位置 D:\\RedisClientForPc\\RedisClient\\Helper\\RelayCommand.cs:行号 62\r\n   在 System.Windows.Input.CommandManager.TranslateInput(IInputElement targetElement, InputEventArgs inputEventArgs)\r\n   在 System.Windows.UIElement.OnMouseDownThunk(Object sender, MouseButtonEventArgs e)\r\n   在 System.Windows.Input.MouseButtonEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)\r\n   在 System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)\r\n   在 System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)\r\n   在 System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)\r\n   在 System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)\r\n   在 System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)\r\n   在 System.Windows.UIElement.RaiseEvent(RoutedEventArgs args, Boolean trusted)\r\n   在 System.Windows.Input.InputManager.ProcessStagingArea()\r\n   在 System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)\r\n   在 System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)\r\n   在 System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)\r\n   在 System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)\r\n   在 System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)\r\n   在 MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)\r\n   在 MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)\r\n   在 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)\r\n   在 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)","FromIP":"No IP Address"}
{"Level":3,"Created":"2018-08-06 15:41:59.986","Description":"Method:TestConnectCommand","Params":[],"ExceptionMsg":"无法连接到redis server!   在 SAEA.RedisSocket.Core.RedisConnection.Connect() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClientLibs\\Redis\\Core\\RedisConnection.cs:行号 108\r\n   在 SAEA.RedisSocket.RedisClient.Connect() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClientLibs\\Redis\\RedisClient.cs:行号 80\r\n   在 RedisClient.ConnectServerDialogViewModel.<>c__DisplayClass21_0.<ExecuteConnect>b__0() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\ConnectServerDialogViewModel.cs:行号 104\r\n   在 System.Threading.Tasks.Task.InnerInvoke()\r\n   在 System.Threading.Tasks.Task.Execute()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.ConnectServerDialogViewModel.<ExecuteConnect>d__21.MoveNext() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\ConnectServerDialogViewModel.cs:行号 101\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.ConnectServerDialogViewModel.<<get_TestCommand>b__20_0>d.MoveNext() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\ConnectServerDialogViewModel.cs:行号 80","FromIP":"No IP Address"}
{"Level":3,"Created":"2018-08-06 17:11:48.111","Description":"Method:DispatcherUnhandledException","Params":[],"ExceptionMsg":"无法连接到redis server!   在 SAEA.RedisSocket.Core.RedisConnection.Connect() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClientLibs\\Redis\\Core\\RedisConnection.cs:行号 108\r\n   在 SAEA.RedisSocket.RedisClient.Connect() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClientLibs\\Redis\\RedisClient.cs:行号 80\r\n   在 RedisClient.RedisClientViewModel.<ExecuteConnect>b__34_0() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 158\r\n   在 System.Threading.Tasks.Task.InnerInvoke()\r\n   在 System.Threading.Tasks.Task.Execute()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<ExecuteConnect>d__34.MoveNext() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 156\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<<get_ConnectCommand>b__30_0>d.MoveNext() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 113\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)\r\n   在 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)\r\n   在 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)","FromIP":"No IP Address"}
{"Level":3,"Created":"2018-08-06 17:21:18.705","Description":"Method:DispatcherUnhandledException","Params":[],"ExceptionMsg":"无法连接到redis server!   在 SAEA.RedisSocket.Core.RedisConnection.Connect() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClientLibs\\Redis\\Core\\RedisConnection.cs:行号 108\r\n   在 SAEA.RedisSocket.RedisClient.Connect() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClientLibs\\Redis\\RedisClient.cs:行号 80\r\n   在 RedisClient.RedisClientViewModel.<ExecuteConnect>b__34_0() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 158\r\n   在 System.Threading.Tasks.Task.InnerInvoke()\r\n   在 System.Threading.Tasks.Task.Execute()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<ExecuteConnect>d__34.MoveNext() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 156\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<<get_ConnectCommand>b__30_0>d.MoveNext() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 113\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)\r\n   在 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)\r\n   在 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)","FromIP":"No IP Address"}
{"Level":3,"Created":"2018-08-06 17:25:30.782","Description":"Method:DispatcherUnhandledException","Params":[],"ExceptionMsg":"“在“System.Windows.Baml2006.TypeConverterMarkupExtension”上提供值时引发了异常。”，行号为“32”，行位置为“56”。   在 System.Windows.FrameworkTemplate.LoadTemplateXaml(XamlReader templateReader, XamlObjectWriter currentWriter)\r\n   在 System.Windows.FrameworkTemplate.LoadTemplateXaml(XamlObjectWriter objectWriter)\r\n   在 System.Windows.FrameworkTemplate.LoadOptimizedTemplateContent(DependencyObject container, IComponentConnector componentConnector, IStyleConnector styleConnector, List`1 affectedChildren, UncommonField`1 templatedNonFeChildrenField)\r\n   在 System.Windows.FrameworkTemplate.LoadContent(DependencyObject container, List`1 affectedChildren)\r\n   在 System.Windows.StyleHelper.ApplyTemplateContent(UncommonField`1 dataField, DependencyObject container, FrameworkElementFactory templateRoot, Int32 lastChildIndex, HybridDictionary childIndexFromChildID, FrameworkTemplate frameworkTemplate)\r\n   在 System.Windows.FrameworkTemplate.ApplyTemplateContent(UncommonField`1 templateDataField, FrameworkElement container)\r\n   在 System.Windows.FrameworkElement.ApplyTemplate()\r\n   在 System.Windows.FrameworkElement.MeasureCore(Size availableSize)\r\n   在 System.Windows.UIElement.Measure(Size availableSize)\r\n   在 System.Windows.Controls.Grid.MeasureOverride(Size constraint)\r\n   在 System.Windows.FrameworkElement.MeasureCore(Size availableSize)\r\n   在 System.Windows.UIElement.Measure(Size availableSize)\r\n   在 System.Windows.Controls.Control.MeasureOverride(Size constraint)\r\n   在 System.Windows.FrameworkElement.MeasureCore(Size availableSize)\r\n   在 System.Windows.UIElement.Measure(Size availableSize)\r\n   在 System.Windows.Controls.Primitives.TabPanel.MeasureOverride(Size constraint)\r\n   在 System.Windows.FrameworkElement.MeasureCore(Size availableSize)\r\n   在 System.Windows.UIElement.Measure(Size availableSize)\r\n   在 System.Windows.ContextLayoutManager.UpdateLayout()\r\n   在 System.Windows.ContextLayoutManager.UpdateLayoutCallback(Object arg)\r\n   在 System.Windows.Media.MediaContext.InvokeOnRenderCallback.DoWork()\r\n   在 System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()\r\n   在 System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)\r\n   在 System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)\r\n   在 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)\r\n   在 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)","FromIP":"No IP Address"}
{"Level":3,"Created":"2018-08-06 19:14:05.329","Description":"Method:DispatcherUnhandledException","Params":[],"ExceptionMsg":"无法连接到redis server!   在 SAEA.RedisSocket.Core.RedisConnection.Connect() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClientLibs\\Redis\\Core\\RedisConnection.cs:行号 108\r\n   在 SAEA.RedisSocket.RedisClient.Connect() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClientLibs\\Redis\\RedisClient.cs:行号 80\r\n   在 RedisClient.RedisClientViewModel.<ExecuteConnect>b__34_0() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 158\r\n   在 System.Threading.Tasks.Task.InnerInvoke()\r\n   在 System.Threading.Tasks.Task.Execute()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<ExecuteConnect>d__34.MoveNext() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 156\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   在 RedisClient.RedisClientViewModel.<<get_ConnectCommand>b__30_0>d.MoveNext() 位置 D:\\MyProjects\\SAEA\\Src\\RedisClient\\ViewModel\\Nodes\\RedisClientViewModel.cs:行号 113\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)\r\n   在 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)\r\n   在 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)","FromIP":"No IP Address"}
